docs/src/tutorials/ode_example.md

```diff
@@ -391,7 +391,48 @@ can also be used for sensitivity analysis, parameter estimation routines,
 and bifurcation plotting. This makes DifferentialEquations.jl a full-stop solution
 for differential equation analysis which also achieves high performance.
 
-## Example 3: Using Other Types for Systems of Equations
+## Example 3: Solving Nonhomogeneous Equations using Parameterized Functions
+Parameterized functions can also be used for building **nonhomogeneous ordinary differential equations** (these are also referred to as ODEs with **nonzero right-hand sides**). They are frequently used as models for dynamical systems with external (in general time-varying) **inputs**. As an example, consider a [model of a pendulum](https://en.wikipedia.org/wiki/Pendulum_(mathematics)) consisting of a slender rod of length `l` and mass `m`:
+
+```math
+\begin{align*}
+\frac{\mathrm{d}\theta(t)}{\mathrm{d}t} &= \omega(t)\\
+\frac{\mathrm{d}\omega(t)}{\mathrm{d}t} &= - \frac{3}{2}\frac{g}{l}\sin\theta(t) + \frac{3}{ml^2}M(t)
+\end{align*},
+```
+where `θ` and `ω` are the angular deviation of the pendulum from the vertical (hanging) orientation and the angular rate, respectively, `M` is an external torque (developed, say, by a wind or a motor), and finally, `g` stands for gravitional acceleration.
+
+```julia
+using DifferentialEquations
+using Plots
+
+l = 1.0                             # length [m]
+m = 1.0                             # mass[m]
+g = 9.81                            # gravitational acceleration [m/s²]
+
+function pendulum!(du,u,p,t)
+    du[1] = u[2]                    # θ'(t) = ω(t)
+    du[2] = -3g/(2l)*sin(u[1]) + 3/(m*l^2)*p(t) # ω'(t) = -3g/(2l) sin θ(t) + 3/(ml^2)M(t)
+end
+
+θ₀ = 0.01                           # initial angular deflection [rad]
+ω₀ = 0.0                            # initial angular velocity [rad/s]
+u₀ = [θ₀, ω₀]                       # initial state vector
+tspan = (0.0,10.0)                  # time interval
+
+M = t->0.1sin(t)                    # external torque [Nm]
+
+prob = ODEProblem(pendulum!,u₀,tspan,M)
+sol = solve(prob)
+
+plot(sol,linewidth=2,xaxis="t",label=["θ [rad]" "ω [rad/s]"],layout=(2,1))
+```
+
+![Pendulum response](../assets/pendulum_response.png)
+
+In particular, note how the external torque `M` is introduced as a *time-varying* parameter in the `pendulum!` function. Indeed, unlike the (vector of) state variables `u`, it must have the dependence on time explicitly expressed.
```

Comment:
we might want to describe this instead as a general principle that the parameters can be any type. Here, we specify `M` as time-varying by representing it by a function.

Reply:
Perhaps the paragraph should be modified into something like this:

Note how the external **time-varying** torque `M` is introduced as a **parameter** in the `pendulum!` function. Indeed, as a general principle the parameters can be any type; here we specify `M` as time-varying by representing it by a function, which is expressed by appending the dependence on time `(t)` to the name of the parameter.  

Note also that, in contrast with the time-varying parameter, the (vector of) state variables `u`, which is generally also time-varying, is always used without the explicit dependence on time `(t)`.

Reply:
yeah, that sounds good!